styles/BlogSingle.module.css

```diff
@@ -0,0 +1,62 @@
+.blog_image_container {
+  width: 1780px;

```

Comment:
We should not define a fixed width, maybe you can think about using a grid here.